README.md

```diff
@@ -459,11 +459,21 @@ Before following these instructions, ensure the existing model you are adding sk
 
    ```shell
    (venv) $ ilab data generate
-   INFO 2024-02-29 19:09:48,804 lab.py:250 Generating model 'ggml-merlinite-7b-lab-Q4_K_M' using 10 CPUs,
-   taxonomy: '/home/username/instructlab/taxonomy' and seed 'seed_tasks.json'
-
-   0%|##########| 0/100 Cannot find prompt.txt. Using default prompt.
-   98%|##########| 98/100 INFO 2024-02-29 20:49:27,582 generate_data.py:428 Generation took 5978.78s
+   INFO 2024-07-30 19:57:44,093 numexpr.utils:161: NumExpr defaulting to 8 threads.
+   INFO 2024-07-30 19:57:44,452 datasets:58: PyTorch version 2.3.1 available.
+   Generating synthetic data using '/root/.cache/instructlab/models/mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf' model, taxonomy:'./taxonomy' against http://localhost:8000/v1 server
```

Comment:
We use `$HOME` elsewhere in the README - is this always using `root`? If not can you update to use `$HOME` instead

Reply:
from what i picked up on in recent conversations things getting stored to root is a problem and shouldn't be happening

Reply:
updated to use $HOME